cmd/deploy/command_configuration.go

```diff
@@ -163,23 +160,24 @@ func addEnvVars(ctx context.Context, cwd string) error {
 		os.Setenv(model.OktetoTokenEnvVar, okteto.Context().Token)
 	}
 	oktetoLog.AddMaskedWord(os.Getenv(model.OktetoTokenEnvVar))
-	return nil
 }
 
-func switchRepoSchemaToHTTPS(repo string) (*url.URL, error) {
+func switchRepoSchemaToHTTPS(repo string) *url.URL {
 	repoURL, err := giturls.Parse(repo)
 	if err != nil {
-		return nil, err
+		return nil
```

Comment:
Might be interesting to include a info or debug with the error to have in our logs but not display it to the user